rust/stackable-cockpit/src/platform/operator/listener_operator.rs

```diff
@@ -0,0 +1,106 @@
+use clap::ValueEnum;
+use snafu::ResultExt;
+use stackable_operator::{
+    k8s_openapi::api::core::v1::Node,
+    kube::{Api, Client, api::ListParams},
+};
+use tokio::sync::OnceCell;
+use tracing::{debug, info, instrument};
+
+pub static LISTENER_OPERATOR_PRESET: OnceCell<ListenerOperatorPreset> = OnceCell::const_new();
+
+#[derive(Copy, Clone, Debug, ValueEnum)]
```

Comment:
Doc comments on pub items

```suggestion
/// Represents the `preset` value in the Listener Operator Helm Chart
#[derive(Copy, Clone, Debug, ValueEnum)]
```